src/main/java/donnafin/logic/commands/SwitchTabCommand.java

```diff
@@ -40,15 +39,8 @@ public SwitchTabCommand(Ui.ViewFinderState tab, PersonAdapter personAdapter) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        Consumer<Ui> uiAction = ui -> {
-            try {
-                ui.switchClientViewTab(tab);
-            } catch (ParseException e) {
-                //This is sufficient as its only an exception due to switch
-                //Handling all cases normally and still having a default
-                e.printStackTrace();
-            }
-        };
+
+        Consumer<Ui> uiAction = ui -> ui.switchClientViewTab(tab);
```

Comment:
just curious why the parse exception no long needs to be caught

Reply:
Whenever `switchClientViewTab` is called, it will use take in an enum to represent the state and there's no real way to pass an invalid argument (which would be the cause of a `ParseException`). Technically you could cause a failure by running switch tab to person list panel, but that's not really a parsing failure. It should be a runtime failure, and it would likely be an assert problem anyhow to be discovered in testing.

So I dropped the invalid argument check that threw a `ParseError`, replaced it with an `assert false : "Illegal enum value."`. Now that its an assertion error, (an unchecked exception), it does not have to be explicitly caught or handled wherever its called.